common/changes/office-ui-fabric-react/jspurlin-ComboBoxExpandOnTouch_2018-03-23-22-33.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added ontouch support to expand menus for split buttons and combo boxes",
```

Comment:
SplitButton/ComboBox: added `onTouch` support for menu expansion.

Reply:
Updated!